modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -27,7 +27,7 @@ link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Share
 * You have completed the AWS prerequisites for ROSA with STS.
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA (`rosa`) and AWS (`aws`) CLIs on your installation host.
+* You have installed and configured the latest {product-title} (ROSA) CLI (`rosa`) and the AWS (`aws`) CLIs on your installation host.
```

Comment:
Same fixes suggested for https://github.com/openshift/openshift-docs/pull/61151/files#r1233482587